app/Http/Controllers/HeadOfProgramController.php

```diff
@@ -33,36 +34,57 @@ public function actives(): Response
                 unset($program->latestEpisode);
                 return $program;
             })->onEachSide(5);
-        dd(json_encode($programs, JSON_PRETTY_PRINT));
-        return Inertia::render('CHANGEME', $programs);
+        return Inertia::render('HeadOfProgram/ActiveProgram', [
+            'programs' => $programs
+        ]);
     }
 
-    public function program(Program $program): Response
+    public function draftProgram(Program $program): Response
     {
-        // TODO: status property
-        $program->episodes = $program->episodes();
-        dd(json_encode($program));
-        return Inertia::render('CHANGEME', $program);
+        if ($program->episodes()->exists()) {
+            abort(404);
+        }
+
+        return Inertia::render('HeadOfProgram/ProgramDetail', [
+            'program' => $program,
+            'source' => 'drafts'
+        ]);
+    }
+
+    public function activeProgram(Program $program): Response
+    {
+        if (!$program->episodes()->exists()) {
+            abort(404);
+        }
+
+        $program->load('episodes');
+
+        return Inertia::render('HeadOfProgram/ProgramDetail', [
+            'program' => $program,
+            'source' => 'actives'
+        ]);
     }
 
     public function create(CreateProgramRequest $req): HttpResponse|ResponseFactory
     {
         try {
             $payload = $req->validated();
             Program::create($payload);
-        } catch (Exception) {
+        } catch (Exception $e) {
+            dd($e->getMessage());
```

Comment:
this is fatal, we dont want to dump the exception information to the user. Its for development purposes only, please remove it

Reply:
okeyy siapp otw🙏🏼